composer.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "pear/spreadsheet_excel_writer",
     "type": "library",
-    "description": "More info available on: http://pear.php.net/package/Spreadsheet_Excel_Writer",
+    "description": "More info available on: https://github.com/pear/Spreadsheet_Excel_Writer",
```

Comment:
```suggestion
    "description": "Spreadsheet_Excel_Writer was born as a porting of the Spreadsheet::WriteExcel Perl module to PHP. It allows writing of Excel spreadsheets without the need for COM objects. It supports formulas, images (BMP) and all kinds of formatting for text and cells.",
```

Reply:
This is quite a long description. I would suggest not repeating the package name and reducing the feature details.

Reply:
Please!